sorc/fv3gfs_build.cfg

```diff
@@ -1,7 +1,7 @@
 #
 # ***** configuration of global-workflow build *****
 
- Building fv3gfs (fv3gfs) .............................. yes
+ Building ufs_model (ufs_model) ........................ yes
```

Comment:
Why retain this build config if we have cpl_build.cfg now?

Reply:
The coupled config doesn't build most of the parts used by GFS (like GSI). Eventually it will, but until then there is no use building parts not needed for the prototypes. There probably should be a separate one for aerosols too (they don't need WW3 pre/post, for example).

Reply:
Yeah, in the future we can rename these files so they are consistent/clearer. We can worry about that later though.